spec/index.html

```diff
@@ -1452,6 +1474,32 @@ <h4>Patterns of RDFS entailment (Informative)</h4>
           </tr>
         </tbody>
       </table>
+      
+      <p/>
+      
+      <table id="rdfs_entailment_patterns_recursive">
+        <tbody>
+          <tr>
+            <th ></th>
+            <th >If the triple or triple term <a>appears in</a> S:</th>
+            <th >then S RDFS <a>entails recognizing D</a>:</th>
+          </tr>
+          <tr >
+            <td class="othertable"><dfn>rdfs4</dfn></td>
+            <td class="othertable">xxx aaa yyy <code></code></td>
+            <td class="othertable">xxx <code>rdf:type rdfs:Resource .</code><br/>
+            yyy <code>rdf:type rdfs:Resource .</code></td>
+          </tr>
+          <tr >
+            <td class="othertable"><dfn>rdfs14</dfn></td>
+            <td class="othertable">xxx yyy &lt;&lt;(aaa bbb ccc)>> <code></code></td>
+            <td class="othertable">the graph S with the appearance replaced with <br/>
```

Comment:
```suggestion
            <td class="othertable">
```
No need for that. RDF is monotonic, so any inferred triple can coexist with the original triples.
In fact, this sentence may give the wrong impression that the original triples must be removed.

Reply:
Sure, but we need to ADD the triples with the bnode substitution. 
On the other hand, we should find a better way to say it.

Reply:
Oh, sorry, now I get it!
The triples on the right are only entailed as is IF the triples on the right are asserted.
But since the rule says they must "appear", then we need to replace their "appearance"...

It is not clear, however, than only the 1st of the 2 triples needs to be "injected". The other one is inferre  asserted.

Also, a number of the triples in the rules are missing the trailing period (`.`).

Reply:
BTW, this raises the issue that entailment patterns in the semantics document are written in Turtle syntax, but probably we should stick to the abstract syntax.